tooling/reproducible/ReproducibleBuilds.md

```diff
@@ -101,7 +150,13 @@ java -XshowSettings:
 ...
 ```
 
-eg.
+In cygwin, you must handle the trailing `\r` otherwise it will fail later:
+
+```bash
+   JAVA_VENDOR="$($JDK_DIR/bin/java -XshowSettings 2>&1| grep 'java.vendor = ' | sed 's/.* = //' | sed 's/\r.*//' )"
+```
+
+eg:
```

Comment:
```suggestion
e.g.,:
```

Reply:
is e.g. really that much preffered over eg: ?   Thanx!